### PR TITLE
ci: try out GitHub actions (upstream)

### PR DESCRIPTION
**Pull request by stephaniewang526**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        java: [7, 8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: test
```
Comment (chingor13) [inline .github/workflows/ci.yaml:21]: Let's remove the coverage build below and add the codecov action like: https://github.com/googleapis/nodejs-asset/blob/master/.github/workflows/ci.yaml#L21-L25

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - run: java -version
+    - run: .kokoro/build.sh
+      env:
+        JOB_TYPE: test
+        REPORT_COVERAGE: true
```
Comment (chingor13) [inline .github/workflows/ci.yaml:62]: Let's remove this one and use the codecov action linked above.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (stephaniewang526) [reply .github/workflows/ci.yaml:21]: done

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (stephaniewang526) [reply .github/workflows/ci.yaml]: done

**Pull request merged**